src/services/wfspermalink.js

```diff
@@ -0,0 +1,304 @@
+goog.provide('ngeo.WfsPermalink');
+
+goog.require('ngeo');
+/** @suppress {extraRequire} - required for `ngeoQueryResult` */
+goog.require('ngeo.Query');
+goog.require('ol.format.WFS');
+
```

Comment:
On the other hand, I worked already when I was working with my own ol3 branch. After that I switched to ol3 master. I will investigate next week.